public/blog/testrpc-is-now-ganache-cli.md

```diff
@@ -0,0 +1,25 @@
+# TestRPC is now Ganache CLI
+
+Have you been looking for the TestRPC recently and haven't been able to find anything about it?
+
+Don't worry, it's still here! It's just now known as **Ganache CLI**. You can get it here:
```

Comment:
Because we want to stay a single brand, I'd say the TestRPC is now Ganache, and it comes in two flavors: A command line version used as a local server and version with a rich user interface. (Aside: you might want to stick with that flavors metaphor when you're actually writing it; feel free to quote me). 

Note you'll likely have to update the section below telling users how to get both if you make this change.

Reply:
I'm struggling a little bit here. If we want both tools to be called Ganache, then shouldn't we mention the CLI version on our [Ganache](http://truffleframework.com/ganache) page, and maybe change the command (and repo name) from `ganache-cli` to just `ganache`? If you go to https://github.com/trufflesuite/ganache-cli right now, it's called "Ganache CLI". It feels like we're being inconsistent.

Branding issues aside, the purpose of this article was specifically to show people how to move from the old command-line tool to the new one. The GUI doesn't feel very important here, and I'd want to reflect that somehow.

Does any of this make sense?